Sources/JExtractSwift/Swift2JavaTranslator+Printing.swift

```diff
@@ -472,30 +440,11 @@ extension Swift2JavaTranslator {
       \(decl.renderCommentSnippet ?? " *")
        */
       public \(parentName.unqualifiedJavaTypeName)(\(renderJavaParamDecls(decl, paramPassingStyle: .wrapper))) {
-        this(/*arena=*/null, \(renderForwardJavaParams(decl, paramPassingStyle: .wrapper)));
```

Comment:
I wonder if we should remove the constructors without an arena passed in actually… fine to keep in this pr, wdyt tho?

Reply:
I'm fine with removing this constructor as we don't want users to use `ofAuto()` arena as much as possible. 
Also mixing different arena in a session sounds scary 😅 E.g.
```Java
try(var arena = SwiftArena.ofConfined()) {
  var obj = MyClass(arena);
  var value = MyStruct(); // Accidental use of "auto" arena.
  obj.process(value); // Probably fine, but...
}
```
This is actually probably fine, but still it's not ideal.
Removing this convenience constructor prevents this kind of issues.

Reply:
Yeah, it’ll be more consistent to just always pass arenas, might save us weird headaches down the line.

you can remove here or we’ll do later, your call

Reply:
Let's do it in this PR 👍